src/main/java/org/mifos/processor/bulk/camel/routes/InitSubBatchRoute.java

```diff
@@ -118,6 +125,18 @@ public void configure() throws Exception {
                     Map<String, Object> variables = exchange.getProperty(ZEEBE_VARIABLE, Map.class);
                     variables.put(PAYMENT_MODE, paymentMode);
                     variables.put(DEBULKINGDFSPID, mapping.getDebulkingDfspid() == null ? tenantName : mapping.getDebulkingDfspid());
+                    if (!(Boolean) variables.get(PARTY_LOOKUP_FAILED)) {
+                        String filename = exchange.getProperty(SERVER_FILE_NAME).toString();
+                        String regex = ".*_sub-batch-([\\w-]+)\\.csv";
```

Comment:
this needs to be added to the documentation. We might miss how payee fsp is transferred